### PR TITLE
Update readme for build binary on ubuntu

### PR DESCRIPTION
**Pull request by ScorpioCPH**
Update readme for build binary on ubuntu, @flx42 PTAL.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -21,6 +23,28 @@ docker run -it -v /var/lib/kubelet/device-plugins:/var/lib/kubelet/device-plugin
 kubectl create -f nvidia-device-plugin.yml
 ```
 
+### Local Env
+
+You `MUST` set environment variable `LIBRARY_PATH` for static linking `cudart_static` library which required [here](https://github.com/NVIDIA/nvidia-docker/blob/master/src/cuda/bindings.go#L5), as show below (Ubuntu 16.04):
```
Comment (flx42) [inline README.md]: This is too complicated. You can just assume that the `/usr/local/cuda` symlink exists.
And then you can do:
```bash
C_INCLUDE_PATH=/usr/local/cuda/include LIBRARY_PATH=/usr/local/cuda/lib64 go build
```

File: README.md [hunk 2]
```diff
@@ -21,6 +23,28 @@ docker run -it -v /var/lib/kubelet/device-plugins:/var/lib/kubelet/device-plugin
...
+```
+
+#### Build Binary
+```shell
+$ go build
+```
+
+#### Run locally
+```shell
+$ ./k8s-device-plugin
```
Comment (flx42) [inline README.md]: You will need `sudo`.
Also remove the `$`, to be consistent with the rest of the README.

File: README.md [hunk 3]
```diff
@@ -21,6 +23,28 @@ docker run -it -v /var/lib/kubelet/device-plugins:/var/lib/kubelet/device-plugin
...
+# 2017/10/19 11:37:22 Loading NVIDIA management library
```
Comment (flx42) [inline README.md]: Remove this line to be consistent.

File: README.md (hunk 1 above)
Comment (ScorpioCPH) [reply README.md]: @flx42 Thanks, good idea!

**Comment by ScorpioCPH** [comment]
@flx42 Addressed your comments, PTAL.

**Comment by flx42** [comment]
Pushed manually after some minor fixes, thanks.

**Pull request closed without merging**